<!DOCTYPE HTML>
<html lang="en">

<head>
  <title>Akhil Kumar</title>

  <meta content="text/html; charset=utf-8" http-equiv="Content-Type">

  <meta name="author" content="Akhil Kumar" />
  <meta name="viewport" content="width=device-width, initial-scale=1">

  <link rel="stylesheet" type="text/css" href="/style.css" />
  <link rel="canonical" href="http://localhost:4000/">
  <link href="https://fonts.googleapis.com/css?family=Lato:400,700,400italic,700italic" rel="stylesheet" type="text/css">

</head>



<body>
  <table style="width:100%;max-width:800px;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;">
    <tr style="padding:0px">
      <td style="padding:0px">
        <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;">
          <tr style="padding:0px">
            <td style="padding:2.5%;width:60%;vertical-align:middle">
              <h1>
                Akhil Kumar
              </h1>
              <p>I am a Chemical Engineering undergraduate from <a href="https://home.iitd.ac.in/">Indian Institute of Technology (IIT) Delhi</a>. I am an Associate Researcher in <a href="https://www.tsankovlab.org/">Tsankov Lab</a> at the <a href="https://icahn.mssm.edu/">Icahn School of Medicine at Mount Sinai</a>, where I leverage single-cell genomics approaches to advance precision medicine for colorectal cancer. Before this, I was a Research Fellow in <a href="https://vperumal.wixsite.com/vivekanandan-perumal">Perumal Lab</a> at the <a href="https://bioschool.iitd.ac.in/">School of Biological Sciences</a> of <a href="https://home.iitd.ac.in/">Indian Institute of Technology (IIT) Delhi</a>, where I used evolutionary bioinformatics approaches to better understand host adaptation dynamics of SARS-CoV-2 and Hepatitis C virus.
              </p>
              <p style="text-align:center">
                <a href="mailto:akhil.kumar@alumni.iitd.ac.in"> Email</a> &nbsp;/&nbsp;
                <a href="https://github.com/iamakhilverma">GitHub</a> &nbsp;/&nbsp;
                <a href="https://scholar.google.com/citations?user=2uOpfM0AAAAJ">Google Scholar</a> &nbsp;/&nbsp;
                <a href="https://www.linkedin.com/in/akhil-kumar-11077212a/"> LinkedIn </a> &nbsp;/&nbsp;
                <a href="/pdfs/cv_akhil_kumar.pdf">CV</a>
              </p>
            </td>
            <td style="padding:2.5%;width:20%;max-width:40%">
              <img style="max-width:100%" alt="profile photo" src="images/profile.png">
            </td>
          </tr>
        </table>

        <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;">
          <tr>
            <td style="padding:2.5%;width:100%;vertical-align:middle">
              <h2>Selected Research</h2>
              <p>
                I'm interested in addressing biomedical problems, particularly in cancer, through functional genomics and systems biology, with an emphasis on computational method development.
              </p>
            </td>
          </tr>
        </table>
        <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;">

          
          <!-- TP53 LUAD paper -->
          <tr>
            <td style="padding:2.5%;width:25%;vertical-align:middle;min-width:120px">
              <img src="/images/research-sinai-at-lab-p53.png" alt="project image" style="width:auto; height:auto; max-width:100%;" />
            </td>
            <td style="padding:2.5%;width:75%;vertical-align:middle">
              <h3>A cellular and spatial atlas of TP53-associated tissue remodeling in lung adenocarcinoma.</h3>
              <br>
              <!-- <em>Perumal Lab, Indian Institute of Technology Delhi</em>
              <br>
              Research Fellow
              <br> -->
              William Zhao*, Thinh T Nguyen*, Atharva Bhagwat*, <strong>Akhil Kumar*</strong>, Bruno Giotti, ..., Aaron N Hata , Aviv Regev , Bruce E Johnson , Alexander M Tsankov
              <!-- William Zhao*, Thinh T Nguyen*, Atharva Bhagwat*, <strong>Akhil Kumar*</strong>, Bruno Giotti, Benjamin Kepecs, Jason L Weirather, Navin R Mahadevan, Asa Segerstolpe, Komal Dolasia, Jamshid Abdul-Ghafar, Naomi R Besson, Stephanie M Jones , Brian Y Soong, Chendi Li , Sebastien Vigneau , Michal Slyper , Isaac Wakiro , Mei-Ju Su , Karla Helvie , Allison Frangieh , Judit Jane-Valbuena , Orr Ashenberg , Mark Awad , Asaf Rotem , Raphael Bueno , Orit Rozenblatt-Rosen , Kathleen Pfaff , Scott Rodig , Aaron N Hata , Aviv Regev , Bruce E Johnson , Alexander M Tsankov -->

              <br>
              <em>Nature Cancer</em>, 2025
              <br>
              
              <a href="https://doi.org/10.1038/s43018-025-01053-7">paper</a> /
                            
              <a href="(https://doi.org/10.5281/zenodo.15866295">code</a> /

              
              <p></p>
              <p>Tumor protein p53 (TP53) is the most frequently mutated gene across many cancers and is associated with shorter overall survival in lung adenocarcinoma (LUAD). Here, to define how TP53 mutations affect the LUAD tumor microenvironment (TME), we constructed a multiomic cellular and spatial atlas of 23 treatment-naive human lung tumors. We found that TP53-mutant malignant cells lose alveolar identity and upregulate highly proliferative and entropic gene expression programs consistently across LUAD tumors from resectable clinical samples, genetically engineered mouse models, and cell lines harboring a wide spectrum of TP53 mutations. We further identified a multicellular tumor niche composed of SPP1+ macrophages and collagen-expressing fibroblasts that coincides with hypoxic, prometastatic expression programs in TP53-mutant tumors. Spatially correlated angiostatic and immune checkpoint interactions, including CD274–PDCD1 and PVR–TIGIT, are also enriched in TP53-mutant LUAD tumors and likely engender a more favorable response to checkpoint blockade therapy. Our systematic approach can be used to investigate genotype-associated TMEs in other cancers.</p>

            </td>
          </tr>

          <!-- COO paper -->
          <tr>
            <td style="padding:2.5%;width:25%;vertical-align:middle;min-width:120px">
              <img src="/images/research-sinai-at-lab-coo.png" alt="project image" style="width:auto; height:auto; max-width:100%;" />
            </td>
            <td style="padding:2.5%;width:75%;vertical-align:middle">
              <h3>Cancer Cell of Origin.</h3>
              <br>
              <!-- <em>Perumal Lab, Indian Institute of Technology Delhi</em>
              <br>
              Research Fellow
              <br> -->
              Mohamad D Bairakdar*, Wooseung Lee*, Bruno Giotti*, <strong>Akhil Kumar*</strong>, Paula Stancl*, Elvin Wagenblast, Dolores Hambardzumyan, Paz Polak, Rosa Karlic, Alexander M Tsankov

              <br>
              <em>Nature communications</em>, 2025
              <br>
              
              <a href="https://doi.org/10.1038/s41467-025-63957-3">paper</a> /
                            
              <a href="https://doi.org/10.5281/ZENODO.16753796">code</a> /

              
              <p></p>
              <p>Deciphering the pre-malignant cell of origin (COO) of different cancers is critical for understanding tumor development and improving diagnostic and therapeutic strategies in oncology. Prior work demonstrates that somatic mutations preferentially accumulate in closed chromatin regions of a cancer’s COO. Leveraging this information, we combine 3,669 whole genome sequencing patient samples, 559 single-cell chromatin accessibility cellular profiles, and machine learning to predict the COO of 37 cancer subtypes with high robustness and accuracy, confirming both the known anatomical and cellular origins of numerous cancers, often at cell subset resolution. Importantly, our data-driven approach predicts a basal COO for most small cell lung cancers and a neuroendocrine COO for rare atypical cases. Our study also highlights distinct cellular trajectories during cancer development of different histological subtypes and uncovers an intermediate metaplastic state during tumorigenesis for multiple gastrointestinal cancers, which have important implications for cancer prevention, early detection, and treatment stratification.</p>

            </td>
          </tr>

          <!-- MBE paper -->
          <tr>
            <td style="padding:2.5%;width:25%;vertical-align:middle;min-width:120px">
              <img src="/images/research-ra-vp-lab-mbe.png" alt="project image" style="width:auto; height:auto; max-width:100%;" />
            </td>
            <td style="padding:2.5%;width:75%;vertical-align:middle">
              <h3>Host-adaptation dynamics in SARS-CoV-2.</h3>
              <br>
              <!-- <em>Perumal Lab, Indian Institute of Technology Delhi</em>
              <br>
              Research Fellow
              <br> -->
              <strong>Akhil Kumar*</strong>, Nishank Goyal*, Nandhini Saranathan, Sonam Dhamija, Saurabh Saraswat, Manoj B Menon, Perumal Vivekanandan

              <br>
              <em>Molecular Biology and Evolution</em>, 2022
              <br>
              
              <a href="https://academic.oup.com/mbe/article/39/3/msac029/6521032">paper</a> /
                            
              <a href="https://github.com/iamakhilverma/analysis_of_sars_cov2_seqs_from_gisaid.git">code</a> /

              
              <p></p>
              <p>Depletion of CpG dinucleotides in severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2) genomes has been linked to virus evolution, host-switching, virus replication, and innate immune responses. Temporal variations, if any, in the rate of CpG depletion during virus evolution in the host remain poorly understood. Here, we analyzed the CpG content of over 1.4 million full-length SARS-CoV-2 genomes representing over 170 million documented infections during the first 17 months of the pandemic. Our findings suggest that the extent of CpG depletion in SARS-CoV-2 genomes is modest. Interestingly, the rate of CpG depletion is highest during early evolution in humans and it gradually tapers off, almost reaching an equilibrium; this is consistent with adaptations to the human host. Furthermore, within the coding regions, CpG depletion occurs predominantly at codon positions 2-3 and 3-1. Loss of ZAP (Zinc-finger antiviral protein)-binding motifs in SARS-CoV-2 genomes is primarily driven by the loss of the terminal CpG within the motifs. Nonetheless, majority of the CpG depletion in SARS-CoV-2 genomes occurs outside ZAP-binding motifs. SARS-CoV-2 genomes selectively lose CpGs-motifs from a U-rich context; this may help avoid immune recognition by TLR7. SARS-CoV-2 alpha-, beta-, and delta-variants of concern have reduced CpG content compared to sequences from the beginning of the pandemic. In sum, we provide evidence that the rate of CpG depletion in virus genomes is not uniform and it greatly varies over time and during adaptations to the host. This work highlights how temporal variations in selection pressures during virus adaption may impact the rate and the extent of CpG depletion in virus genomes.</p>

            </td>
          </tr>
          
          
          
          
          
          
          
        </table>
        <br>
        <br>
        <br>
        <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;">

          <tr>
            <td style="padding:2.5%;width:100%;vertical-align:middle">
              <h2>Work Under Review</h2>
              <p> Recent findings currently submitted for peer review. 
              </p>
            </td>
          </tr>
        </table>
        <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;">


          <!-- under-review paper -->
          <tr>
            <td style="padding:2.5%;width:25%;vertical-align:middle;min-width:120px">
              <img src="/images/research-ra-vp-lab-under_review.png" alt="project image" style="width:auto; height:auto; max-width:100%;" />
            </td>
            <td style="padding:2.5%;width:75%;vertical-align:middle">
              <h3>Early mutation blitzkrieg in SARS-CoV-2.</h3>
              <br>
              <!-- <em>Perumal Lab, Indian Institute of Technology Delhi</em>
              <br>
              Research Fellow
              <br> -->
              <strong>Akhil Kumar*</strong>, Rishika Kaushal*, Manoj B Menon, Vivekanandan Perumal

              <br>
              <em>Under-review</em>
              <br>
              
              <!-- <a href="https://academic.oup.com/mbe/article/39/3/msac029/6521032">paper</a> / -->
                            
              <a href="https://github.com/iamakhilverma/early_mutation_blitzkrieg_in_sars_cov2.git">code</a> /

              
              <p></p>
              <p>Analysis of 16.7 million SARS-CoV-2 genomes to understand mutation emergence patterns during viral evolution.</p>

            </td>
          </tr>
          
          
          
        </table>
        <br>
        <br>
        <br>
        <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;">

          <tr>
            <td style="padding:2.5%;width:100%;vertical-align:middle">
              <h2>Additional Research</h2>
              <p> A collection of other publications and prior projects.
              <!-- <p> These include published secondary projects, and unpublished research work from undergraduate time (including internships, thesis, and independent studies).  -->
              </p>
            </td>
          </tr>
        </table>
        <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;">


          <!-- Conserved stretches paper -->
          <tr>
            <td style="padding:2.5%;width:25%;vertical-align:middle;min-width:120px">
              <img src="/images/research-ra-vp-lab-bifg.png" alt="project image" style="width:auto; height:auto; max-width:100%;" />
            </td>
            <td style="padding:2.5%;width:75%;vertical-align:middle">
              <h3>Improvement of diagnostic assays against emerging variants of SARS-CoV-2.</h3>
              <br>
              <!-- <em>Perumal Lab, Indian Institute of Technology Delhi</em>
              <br>
              Research Fellow
              <br> -->
              <strong>Akhil Kumar</strong>, Rishika Kaushal, Himanshi Sharma, Khushboo Sharma, Manoj B Menon, Vivekanandan Perumal

              <br>
              <em>Briefings in Functional Genomics</em>, 2024
              <br>
              
              <a href="https://doi.org/10.1093/bfgp/elad027">paper</a> /
                            
              <a href="https://github.com/iamakhilverma/conserved_stretches_in_sars_cov2.git">code</a> /

              
              <p></p>
              <p>We identified 11 conserved stretches in over 6.3 million SARS-CoV-2 genomes including all the major variants of concerns. Each conserved stretch is ≥100 nucleotides in length with ≥99.9% conservation at each nucleotide position. Interestingly, six of the eight conserved stretches in ORF1ab overlapped significantly with well-folded experimentally verified RNA secondary structures. Furthermore, two of the conserved stretches were mapped to regions within the S2-subunit that undergo dynamic structural rearrangements during viral fusion. In addition, the conserved stretches were significantly depleted for zinc-finger antiviral protein (ZAP) binding sites, which facilitated the recognition and degradation of viral RNA. These highly conserved stretches in the SARS-CoV-2 genome were poorly conserved at the nucleotide level among closely related β-coronaviruses, thus representing ideal targets for highly specific and discriminatory diagnostic assays. Our findings highlight the role of structural constraints at both RNA and protein levels that contribute to the sequence conservation of specific genomic regions in SARS-CoV-2.</p>

            </td>
          </tr>

          <!-- Pathogens HCV paper -->
          <tr>
            <td style="padding:2.5%;width:25%;vertical-align:middle;min-width:120px">
              <img src="/images/research-ra-vp-lab-hcv.png" alt="project image" style="width:auto; height:auto; max-width:100%;" />
            </td>
            <td style="padding:2.5%;width:75%;vertical-align:middle">
              <h3>Host-adaptation dynamics in Hepatitis C virus.</h3>
              <br>
              <!-- <em>Perumal Lab, Indian Institute of Technology Delhi</em>
              <br>
              Research Fellow
              <br> -->
              Sanket Mukherjee*, <strong>Akhil Kumar*</strong>, Jasmine Samal, Ekta Gupta, Vivekanandan Perumal, Manoj B Menon

              <br>
              <em>Pathogens</em>, 2022
              <br>
              
              <a href="https://doi.org/10.3390/pathogens12010043">paper</a> /
                            
              <a href="https://github.com/iamakhilverma/hcv_seqs_analysis.git">code</a> /

              
              <p></p>
              <p>Hepatitis C virus (HCV) is a bloodborne pathogen that can cause chronic liver disease and hepatocellular carcinoma. The loss of CpGs from virus genomes allows escape from restriction by the host zinc-finger antiviral protein (ZAP). The evolution of HCV in the human host has not been explored in the context of CpG depletion. We analysed 2616 full-length HCV genomes from 1977 to 2021. During the four decades of evolution in humans, we found that HCV genomes have become significantly depleted in (a) CpG numbers, (b) CpG O/E ratios (i.e., relative abundance of CpGs), and (c) the number of ZAP-binding motifs. Interestingly, our data suggests that the loss of CpGs in HCV genomes over time is primarily driven by the loss of ZAP-binding motifs; thus suggesting a yet unknown role for ZAP-mediated selection pressures in HCV evolution. The HCV core gene is significantly enriched for the number of CpGs and ZAP-binding motifs. In contrast to the rest of the HCV genome, the loss of CpGs from the core gene does not appear to be driven by ZAP-mediated selection. This work highlights CpG depletion in HCV genomes during their evolution in humans and the role of ZAP-mediated selection in HCV evolution.</p>

            </td>
          </tr>
          

          <tr>
            <td style="padding:2.5%;width:25%;vertical-align:middle;min-width:120px">
              <img src="/images/research-iit-vp-lab-amtdb.png" alt="project image" style="width:auto; height:auto; max-width:100%;" />
            </td>
            <!-- <td style="padding:2.5%;width:25%;vertical-align:middle;min-width:120px">
              <img src="/tn/images/research-iit-vp-lab-amtdb.png" alt="project image" style="width:auto; height:auto; max-width:100%;" />
            </td> -->
            <td style="padding:2.5%;width:75%;vertical-align:middle">
              <h3>Temporal evolution of mono- and dinucleotides in human mitochondrial DNA</h3>
              <br>
              <em>Perumal Lab, Indian Institute of Technology Delhi</em>
              <br>
              <em>Independent study</em>
              <br>
              2019-11-28

              <br>
              
              
              
              
              
              
              <a href="/pdfs/slides-iit-vp-lab-amtdb.pdf">slides</a> /
              
              
              
              
              
              
              
              
              <p></p>
              <p>Inspected temporal evolution of mono- and dinucleotides in human mtDNA by analysing sequences dated back to 50k BC. Developed statistical frameworks to compare compositional shifts across time periods using curated ancient DNA datasets. Identified a strand-specific compositional asymmetry with GC-rich ancient reads, indicating a directional mutational bias.</p>
              <!-- <p>Inspected temporal evolution of mono- and dinucleotides in human mtDNA by analysing sequences dated back to 50k BC. Developed boxplots for samples from earliest-, middle- and latest-period; conducted statistical tests to compare their medians. Our findings suggested that lighter strand is becoming heavier, and a slight bias toward GC-rich sequences in ancient reads.</p> -->

            </td>
          </tr>
          
          
          
          
          
          <tr>
            <td style="padding:2.5%;width:25%;vertical-align:middle;min-width:120px">
              <img src="/images/research-iit-gg-lab-struc-bioinfo.png" alt="project image" style="width:auto; height:auto; max-width:100%;" />
            </td>
            <!-- <td style="padding:2.5%;width:25%;vertical-align:middle;min-width:120px">
              <img src="/tn/images/research-iit-gg-lab-struc-bioinfo.png" alt="project image" style="width:auto; height:auto; max-width:100%;" />
            </td> -->
            <td style="padding:2.5%;width:75%;vertical-align:middle">
              <h3>In-silico design of small ligand molecules to inhibit early-stage insulin aggregation nucleation</h3>
              <br>
              <em>Multiscale Modeling Group, Indian Institute of Technology Delhi</em>
              <br>
              <em>Coursework</em>
              <br>
              2019-11-24

              <br>
              
              
              
              
              
              
              <a href="/pdfs/slides-iit-gg-lab-struc-bioinfo.pdf">slides</a> /
              
              
              
              
              
              
              
              
              <p></p>
              <p>Assisted with the in silico design of small ligand molecules aimed at inhibiting early-stage insulin aggregation nucleation. Identified EGCG and polyoxometalates as candidate ligands through extensive literature review and computational modeling. Performed targeted docking of EGCG into a partially folded insulin intermediate using the Gray lab’s ROSIE server.</p>
              <!-- <p>Assisted with the in-silico design of small ligand molecules to inhibit early-stage insulin aggregation nucleation. Identified EGCG and polyoxometalates as the putative ligands through extensive literature review, programmed their structures. Performed targeted docking of EGCG into a partially folded intermediate of insulin using Gray lab’s ROSIE server.</p> -->

            </td>
          </tr>
          
          
          
          
          
          <tr>
            <td style="padding:2.5%;width:25%;vertical-align:middle;min-width:120px">
              <img src="/images/research-blr-sys-bio.png" alt="project image" style="width:auto; height:auto; max-width:100%;" />
            </td>
            <!-- <td style="padding:2.5%;width:25%;vertical-align:middle;min-width:120px">
              <img src="/tn/images/research-blr-sys-bio.png" alt="project image" style="width:auto; height:auto; max-width:100%;" />
            </td> -->
            <td style="padding:2.5%;width:75%;vertical-align:middle">
              <h3>Systematic construction and validation of kinetic model from metabolic networks</h3>
              <br>
              <em>Biomolecular Computational Group, Indian Institute of Science Bangalore</em>
              <br>
              <em>Summer Internship</em>
              <br>
              2019-07-15

              <br>
              
              
              
              
              
              
              <a href="/pdfs/slides-blr-sys-bio.pdf">slides</a> /
              
              
              
              
              
              
              
              
              <p></p>
              <p>Evaluated the existing kinetic modeling approaches to translate metabolic networks of interest into a dynamic model. Designed a convenience kinetics–based dynamic model comprising 92 reactions, 110 species using complex pathway simulator. Distributed the reactions across 4 compartments and simulated its dynamic characterstics; examined robustness of the model.</p>
              <!-- <p>Investigated existing kinetic modelling approaches to translate metabolic networks of interest into a dynamic model. Designed a convenience kinetics based dynamic model comprising 92 reactions, 110 species using complex pathway simulator. Distributed the reactions across 4 compartments and simulated its dynamic characterstics; examined robustness of the model.</p> -->

            </td>
          </tr>
          
          
          
          
          
          <tr>
            <td style="padding:2.5%;width:25%;vertical-align:middle;min-width:120px">
              <img src="/images/research-ox-scrna-seq.png" alt="project image" style="width:auto; height:auto; max-width:100%;" />
            </td>
            <!-- <td style="padding:2.5%;width:25%;vertical-align:middle;min-width:120px">
              <img src="/tn/images/research-ox-scrna-seq.png" alt="project image" style="width:auto; height:auto; max-width:100%;" />
            </td> -->
            <td style="padding:2.5%;width:75%;vertical-align:middle">
              <h3>Single-cell transcriptomic data analysis</h3>
              <br>
              <em>Srinivas Group, University of Oxford</em>
              <br>
              <em>Summer Internship</em>
              <br>
              2018-07-24

              <br>
              
              
              
              
              <a href="https://github.com/iamakhilverma/2018-oxford-scrna-seq-analysis.git">code</a> /
              
              
              
              
              
              
              
              
              
              
              <p></p>
              <p>Analysed single-cell transcriptomic profiles collected from early mouse embryos carrying a mutation in the ASPP2 gene. Processed sequence data files, mapped sequence reads, performed quality control on the individual cells, normalized the data. Identified distinct cell subtypes using an unbiased hierarchical clustering algorithm and delineated developmental heterogeneity.
<br /><br />NOTE: The code repository for this project is private, please contact me for access.</p>
              <p>Analysed single-cell transcriptomic profiles collected from early mouse embryos carrying a mutati<!-- on in the ASPP2 gene. Processed sequence data files, mapped sequence reads, performed quality control on the individual cells, normalized the data. Identified distinct cell subtypes using an unbiased hierarchical clustering algorithm and delineated developmental heterogeneity.
              <br /><br />NOTE: The code repository for this project is private, please contact me for access.</p> -->

            </td>
          </tr>
          
          
          
        </table>

        <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;">
          <tr>
            <td style="padding:2.5%;width:100%;vertical-align:middle">
              <h2>Blog</h2>
              <p>
                Coming soon.
              </p>
            </td>
          </tr>
        </table>
        <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;">

          
          
          
          
          
          
          
          
          
          
          
          
          
          
          
          
          
          
          
          
          
        </table>

        <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;">
          <tr>
            <td style="padding:0px">
              <br>
              <p style="text-align:center;font-size:small;">
                Design and source code from <a href="https://leonidk.com/">Leonid Keselman</a>'s <a href="https://github.com/leonidk/new_website">Jekyll fork</a> of <a style="font-size:small;" href="https://jonbarron.info">Jon Barron's website</a>.
              </p>
            </td>
              </p>
            </td>
          </tr>
        </table>
      </td>
    </tr>
  </table>
</body>

</html>

